locales/de.json

```diff
@@ -0,0 +1,96 @@
+{
+	"File": "Datei",
+	"Quit": "Schliessen",
+	"Edit": "Bearbeiten",
+	"Undo": "Lösen",
+	"Redo": "Noch einmal",
+	"Cut": "Schneiden",
+	"Copy": "Kopieren",
+	"Paste": "Einfügen",
+	"Delete": "Löschen",
+	"Select all": "Alles markieren //?",
+	"View": "Wiedergabe",
+	"Reload": "Neu laden",
+	"Force Reload": "Neu laden // erzwingen?",
+	"Toggle Developer Tools": "Entwicklerwerkzeuge umschalten",
+	"Actual size": "Wahre Größe",
```

Comment:
@PrestonN Which context do we have here?

Reply:
I managed to open the menu xD If I understand this correctly, you want to zoom to 100%. Maybe we should rename this, to make this clear. Otherwise, I'd translate it with `Normale Größe`.